sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/application-restriction-bypass.aest.controller.ts

```diff
@@ -50,4 +90,153 @@ export class ApplicationRestrictionBypassAESTController extends BaseController {
 
     return { bypasses };
   }
+
+  /**
+   * Gets application restriction bypass for a given application restriction bypass id.
+   * @param id id of the application restriction bypass.
+   * @returns application restriction bypass.
+   */
+  @ApiNotFoundResponse({
+    description: "Application restriction bypass not found.",
+  })
+  @Get(":id")
+  async getApplicationRestrictionBypass(
+    @Param("id", ParseIntPipe) id: number,
+  ): Promise<ApplicationRestrictionBypassAPIOutDTO> {
+    const applicationRestrictionBypass =
+      await this.applicationRestrictionBypassService.getApplicationRestrictionBypass(
+        id,
+      );
+    if (!applicationRestrictionBypass) {
+      throw new NotFoundException(APPLICATION_RESTRICTION_BYPASS_NOT_FOUND);
```

Comment:
Please follow the other existing `NotFoundException` and throw a message instead of the error code.